test/c/CMakeLists.txt

```diff
@@ -32,12 +31,7 @@ target_link_libraries (test_driver ${qiskit} common)
 if (MSVC)
     # Store the directory where the qiskit_cext.dll is located.
     get_filename_component(qiskit_dll_dir ${qiskit} DIRECTORY)
-
-    # Get the Python runtime library, which has been stored into a config file during the 
-    # crate build process. This is to ensure we link against the same library that PyO3 is using.
-    file(READ "${CMAKE_SOURCE_DIR}/target/pyo3_python.config" pyo3_config)
```

Comment:
Does the `pyo3_python.config` file still have value if we don't use it here? We could also remove it (fine to do as follow up) if we don't need it anymore

Reply:
I removed this in: https://github.com/Qiskit/qiskit/pull/14239